internal/services/containers/kubernetes_nodepool.go

```diff
@@ -198,6 +198,7 @@ func SchemaDefaultNodePool() *pluginsdk.Schema {
 						Optional: true,
 						Computed: true, // defaults to Ubuntu if using Linux
 						ValidateFunc: validation.StringInSlice([]string{
+							string(agentpools.OSSKUAzureLinux),
```

Comment:
Could you add a test for this as well?

Reply:
Sure. I'll do my best to have that added to day.

Reply:
The test has been added.